rules/no-mod-iter-var-for-loop.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @fileoverview Flag for loops which modify their iteration variable in their body
+ * @author Nicolas Feignon <nfeignon@gmail.com>
+ */
+
+'use strict';
+
+module.exports = {
+
+    meta: {
+
+        docs: {
+            recommended: true,
+            type: 'error',
+            description: 'Flag for loops which modify their iteration variable in their body'
+        },
+
+        schema: []
+
+    },
+
+    create: function (context) {
+
+        function inspectLoopStatement(emitted) {
+            var node = emitted.node;
+            if (!emitted.exit) {
+                if (!node.init || node.init.type !== 'AssignmentExpression')
+                    return;
```

Comment:
even when you only have a single statement inside the `if` block, please include it inside `{}`